.circleci/config.yml

```
@@ -603,78 +359,196 @@ workflows:
               ignore:
                 - /release-[0-9]+\.[0-9]+.*/ # match with releaseX.Y.*
 
-
-      - build-11
-      - build-12
-      - build-13
+      - build:
+          name: build-11
+          postgres: 11
+          tag: 11.10-dev202012151914
+      - build:
+          name: build-12
+          postgres: 12
+          tag: 12.5-dev202012151914
+      - build:
+          name: build-13
+          postgres: 13
+          tag: 13.1-dev202012151914
 
       - check-style
       - check-sql-snapshots
 
-      - test-11_check-multi:
+      - test-citus:
+          name: 'test-11_check-multi'
+          postgres: 11
+          tag: 11.10-dev202012151914
+          make: check-multi
           requires: [build-11]
-      - test-11_check-vanilla:
+      - test-citus:
+          name: 'test-11_check-multi-mx'
+          postgres: 11
+          tag: 11.10-dev202012151914
+          make: check-multi-mx
           requires: [build-11]
-      - test-11_check-isolation:
+      - test-citus:
+          name: 'test-11_check-vanilla'
+          postgres: 11
+          tag: 11.10-dev202012151914
+          make: check-vanilla
           requires: [build-11]
-      - test-11_check-mx:
+      - test-citus:
+          name: 'test-11_check-isolation'
+          postgres: 11
+          tag: 11.10-dev202012151914
+          make: check-isolation
           requires: [build-11]
-      - test-11_check-worker:
+      - test-citus:
+          name: 'test-11_check-worker'
+          postgres: 11
+          tag: 11.10-dev202012151914
+          make: check-worker
           requires: [build-11]
-      - test-11_check-follower-cluster:
+      - test-citus:
+          name: 'test-11_check-follower-cluster'
+          postgres: 11
+          tag: 11.10-dev202012151914
+          make: check-follower-cluster
           requires: [build-11]
-      - test-11_check-failure:
+      - test-citus:
+          name: 'test-11_check-failure'
+          postgres: 11
+          image: citus/failtester
+          tag: 11.10-dev202012151914
+          make: check-failure
           requires: [build-11]
 
-      - test-12_check-multi:
+      - test-citus:
+          name: 'test-12_check-multi'
+          postgres: 12
+          tag: 12.5-dev202012151914
+          make: check-multi
           requires: [build-12]
-      - test-12_check-vanilla:
+      - test-citus:
+          name: 'test-12_check-multi-mx'
```

revert back to old name of test. that will prevent us from having to change github settings

```suggestion
          name: 'test-12_check-mx'
```